qemu/tests/nic_hotplug.py

```diff
@@ -97,6 +96,20 @@ def ping_hotplug_nic(ip, mac, session, is_linux_guest):
             status, output = session.cmd_output_safe(del_route_cmd)
         return status, output
 
+    def device_add_nic(pci_model="virtio", netdev="hostnet1", device_id="net1"):
```

Comment:
It's invalid parameter test, I don't think it is necessary to get value from.. such as cfg file or something like this.
All I need is: add a nic; wait qemu to tell me: wrong operation.
Only the error message qemu give me is what matters.
netdev, device_id could be set to default value AND these are not important in this case, only error message matters.